targets/TARGET_Silicon_Labs/TARGET_EFM32/TARGET_EFR32FG1/device/TOOLCHAIN_GCC_ARM/efr32fg1p.ld

```diff
@@ -0,0 +1,216 @@
+/* Linker script for Silicon Labs EFR32FG1P devices                      */
+/*                                                                     */
+/* This file is subject to the license terms as defined in ARM's       */
+/* CMSIS END USER LICENSE AGREEMENT.pdf, governing the use of          */
+/* Example Code.                                                       */
+/*                                                                     */
+/* Copyright 2017 Silicon Laboratories, Inc. http://www.silabs.com     */
+/*                                                                     */
+/* Version 5.1.3 */
+/*                                                                     */
+
+
+MEMORY
+{
+  FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 1024k
```

Comment:
EFR32FG1 has 256k of flash and 32k of ram...

Reply:
Whoops, didn't notice the switch between the PG1B and PG12B.